gen_cert.py

```diff
@@ -102,7 +102,9 @@ class CertificateGen(object):
     """Manages the pdf, signatures, and S3 bucket for course certificates."""
 
     def __init__(self, course_id, template_pdf=None, aws_id=None, aws_key=None,
-                 dir_prefix=None, long_org=None, long_course=None, issued_date=None):
+                 dir_prefix=None, long_org=None, long_course=None, issued_date=None,
+        title=None,
```

Comment:
Yep, this also showed up on Travis testing.  The three pep8 errors I introduced were lost in the noise of the other 70.  Unfortunately, most of those were introduced by our team.